src/Project.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {Builder as _Builder} from './Project/Builder';
+
+export namespace Project {
+
+// eslint-disable-next-line @typescript-eslint/naming-convention
```

Comment:
Just curiosity, what is it for?

Reply:
lint will try to change `Builder` to `builder`. it uses camel case that starts with lower case.